pom.xml

```diff
@@ -95,7 +95,7 @@
             <dependency>
                 <artifactId>solr-solrj</artifactId>
                 <groupId>org.apache.solr</groupId>
-                <version>7.7.3</version>
+                <version>8.2.0</version>
```

Comment:
org.springframework.boot.actuate.solr.SolrHealthIndicator from spring-boot-actuator:2.2.x is not compatible with solr-solrj:7.7.3
It is compiled against solrj:8.2.0 and fails with 7.7.3 trying to use non-existent constructor